.github/workflows/go_generate_update.yml

```diff
@@ -19,29 +25,48 @@ jobs:
                 with:
                     go-version-file: 'go.mod'
             -
-                name: Prepare
+                name: Prepare, generate and format code
                 run: |
                     rm -rf ~/.platformsh/bin/
-                    go generate ./
+                    go generate ./...
+                    go fmt ./...
```

Comment:
We need to exclude generated Go files (like `local/platformsh/commands.go`, `local/platformsh/config.go`)

Reply:
actually I would say the opposite but those files are annoyingly not properly formatted so each time un run `go fmt` I end up with local changes to discard :/

Reply:
but I just though: or we could just fix the generator to have the generated file in a "clean" state.
I'm going to open another PR for this

Reply:
see #576